<Xcpt>
  <Time>24.04.2017 13:24:27</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>24.04.2017 13:24:45</Time>
  <Info>id: 7</Info>
</Info>
<Info>
  <Time>24.04.2017 13:24:45</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:08</Time>
  <Info>id: 2</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:08</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:09</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 436</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 619</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 564</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 689</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 170</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 588</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 460</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 184</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 623</Info>
</Info>
<Info>
  <Time>24.04.2017 13:25:10</Time>
  <Info>Card: 287</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:34</Time>
  <Info>id: 12</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:34</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:53</Time>
  <Info>id: 9</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:53</Time>
  <Info>numPacks: 1</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>vID: 382</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 470</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 69</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 636</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 681</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 19</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 529</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 614</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 296</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 419</Info>
</Info>
<Info>
  <Time>24.04.2017 13:39:54</Time>
  <Info>Card: 101</Info>
</Info>
<Info>
  <Time>24.04.2017 18:19:57</Time>
  <Info>Entered Pass = Thomas</Info>
</Info>
<Info>
  <Time>24.04.2017 18:19:57</Time>
  <Info>HashPass = A41992D2AB9EEF6458B335A1892773E3F0338451563B0D3ACA06327E4494D21D92AF3F4D94BC3C5D66DF4EC9F45C528F91333A60BAE7E707A34047645F912CFB</Info>
</Info>
<Xcpt>
  <Time>24.04.2017 18:19:57</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>24.04.2017 19:46:33</Time>
  <Info>Entered Pass = Thomas</Info>
</Info>
<Info>
  <Time>24.04.2017 19:46:33</Time>
  <Info>HashPass = A41992D2AB9EEF6458B335A1892773E3F0338451563B0D3ACA06327E4494D21D92AF3F4D94BC3C5D66DF4EC9F45C528F91333A60BAE7E707A34047645F912CFB</Info>
</Info>
<Xcpt>
  <Time>24.04.2017 19:46:33</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>24.04.2017 19:48:00</Time>
  <Info>id: 7</Info>
</Info>
<Info>
  <Time>24.04.2017 19:48:00</Time>
  <Info>numPacks: 1</Info>
</Info>
<Xcpt>
  <Time>24.04.2017 21:12:51</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>24.04.2017 21:13:26</Time>
  <Info>GetNumUsers 17</Info>
</Info>
<Info>
  <Time>24.04.2017 21:13:26</Time>
  <Info>GetNumCards 382</Info>
</Info>
<Info>
  <Time>24.04.2017 21:13:26</Time>
  <Info>GetNumDecks 1</Info>
</Info>
<Xcpt>
  <Time>24.04.2017 21:15:48</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 21:25:12</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 21:43:21</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 21:44:01</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 21:52:51</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 21:53:30</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 22:56:16</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 22:57:47</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>24.04.2017 22:59:55</Time>
  <Info>Entered Pass = Franz</Info>
</Info>
<Info>
  <Time>24.04.2017 22:59:55</Time>
  <Info>HashPass = 30E098CD8BC8857FB5033C6AD1B42A8C7EF349D75FEEE119E7E03FC67039E6767F5482F08187013D3CC56550202381B78C636A0EE8026175E610AD2E1C6F338E</Info>
</Info>
<Info>
  <Time>24.04.2017 23:00:00</Time>
  <Info>Entered Pass = franz</Info>
</Info>
<Info>
  <Time>24.04.2017 23:00:00</Time>
  <Info>HashPass = 988D5CC169530C12B7B826C60CB23C3267AE3D8A24ECD91FF3BD67E98588AF0049D1FA52A8E7D3060F3F10E7C794D842AAD76B06C179C60BED5E01F18C25651A</Info>
</Info>
<Xcpt>
  <Time>24.04.2017 23:13:03</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Info>
  <Time>24.04.2017 23:13:25</Time>
  <Info>Entered Pass = Franz</Info>
</Info>
<Info>
  <Time>24.04.2017 23:13:25</Time>
  <Info>HashPass = 30E098CD8BC8857FB5033C6AD1B42A8C7EF349D75FEEE119E7E03FC67039E6767F5482F08187013D3CC56550202381B78C636A0EE8026175E610AD2E1C6F338E</Info>
</Info>
<Info>
  <Time>24.04.2017 23:13:32</Time>
  <Info>Entered Pass = franz</Info>
</Info>
<Info>
  <Time>24.04.2017 23:13:32</Time>
  <Info>HashPass = 988D5CC169530C12B7B826C60CB23C3267AE3D8A24ECD91FF3BD67E98588AF0049D1FA52A8E7D3060F3F10E7C794D842AAD76B06C179C60BED5E01F18C25651A</Info>
</Info>
<Xcpt>
  <Time>24.04.2017 23:38:48</Time>
  <Exception>
    <Source>mscorlib</Source>
    <Message>Ein Element mit dem gleichen Schlüssel wurde bereits hinzugefügt.</Message>
    <Stack>   bei System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   bei System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   bei CardGame.DAL.Logic.UserManager..cctor() in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\UserManager.cs:Zeile 45.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 23:41:53</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User-Emailadresse gibt es bereits</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.Register(tblperson regUser) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 31.</Stack>
  </Exception>
</Xcpt>
<Xcpt>
  <Time>24.04.2017 23:43:08</Time>
  <Exception>
    <Source>CardGame.DAL</Source>
    <Message>User-Emailadresse gibt es bereits</Message>
    <Stack>   bei CardGame.DAL.Logic.AuthManager.Register(tblperson regUser) in C:\Users\FaulThomas\Documents\GitHub\Faul_WebShop_LAP\CardGame\CardGame\CardGame.DAL\Logic\AuthManager.cs:Zeile 31.</Stack>
  </Exception>
</Xcpt>
